src/Draco.Compiler/Internal/Symbols/FunctionSymbol.cs

```diff
@@ -111,6 +114,25 @@ public override string ToString()
         return result.ToString();
     }
 
+    public override bool SignatureEquals(Symbol other)
```

Comment:
I think we are checking too much here. We do eventually need to check things like same "staticness" or same visibility, but we first need to pair up symbols without them. Otherwise we can't tell the user "you meant to override X but it's incorrect".